tred_refactored/devel/unix_install/packages_unix/install.cfg

```
@@ -185,6 +177,7 @@ sub version_from {
     'IO-Compress-' => 'IO::Compress::Base',
     'PDF-API2-' => 'AREIBENS/PDF-API2-0.73.tar.gz',
     'libwww-perl-' => 'OALDERS/libwww-perl-6.77.tar.gz',
+    'Tk-CodeText-' => 'HANJE/Tk-CodeText-0.3.4.tar.gz', # new versions are not working on MacOS due to Imager-Screenshot dependency (scdarwin2.c:61:22: error: 'CGDisplayCreateImageForRect' is unavailable: obsoleted in macOS 15.0)
```

It would be great to set your Mac as a CPAN Tester. There are no failures for [darwin](http://matrix.cpantesters.org/?dist=Imager-Screenshot%200.014;os=darwin;reports=1) at the Testers page. I can help you with that, but it should be easily googlable.

Also, TrEd works happily even without Tk::CodeText at all. I don't have it installed and I see no problems with that - I don't get syntax highlighting when writing code in the Alt+E window, but

---

A nešlo by tu změnu podmínit 
```suggestion
'Tk-CodeText-' => ('darwin' eq $^O) ? 'HANJE/Tk-CodeText-0.3.4.tar.gz' : 'Tk::CodeText',
```
?